localization/en/data.json

```diff
@@ -456,6 +456,11 @@
         "name": "Minimal query length",
         "hint": "for global search",
         "text": "at least 4 characters"
+      },
```

Comment:
If you move items between sections, add, or remove them, you need to update `/data/structore.json` too